utilities/docker/supervisord.conf

```diff
@@ -0,0 +1,11 @@
+[supervisord]
+nodaemon=true
+
+[program:ovn-nb-tcp]
+command=/bin/start-ovn ovn-nb-tcp
+
+[program:ovn-sb-tcp]
+command=/bin/start-ovn ovn-sb-tcp
+
+[program:ovn-northd-tcp]
+command=/bin/start-ovn ovn-northd-tcp
```

Comment:
It seems we don't need northd for go-ovn testing.

Reply:
correct, but convenient to have a running container built from this image that has all the relevant services running